src/rp/config/WheeledRobotConfiguration.java

```diff
@@ -81,6 +89,38 @@ public WheeledRobotConfiguration(float _wheelDiameter, float _trackWidth,
 		};
 	}
 
+	public WheeledRobotConfiguration(float _wheelDiameter, float _trackWidth,
+			float _robotLength, RegulatedMotor _leftWheel,
+			RegulatedMotor _rightWheel, boolean _reversed) {
+		super();
+		m_wheelDiameter = _wheelDiameter;
```

Comment:
Duplication is bad! Instead of copying the whole constructor, make the previous constructor call this one but supply false for the final argument.